src/ui/components/common/icon.rs

```diff
@@ -34,19 +35,23 @@ pub enum IconType {
   Home,
   Profile,
   Hamburger,
+  Users,
+  Posts,
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum IconSize {
   Normal,
   Large,
+  ExtraLarge,
   Small,
 }
 
 impl IntoAttribute for IconSize {
   fn into_attribute(self) -> Attribute {
     match self {
-      Self::Large => "3rem",
+      Self::ExtraLarge => "3rem",
+      Self::Large => "2.25rem",
       Self::Small => "0.75rem",
       _ => "1.5rem",
     }
```

Comment:
If I find myself needing another size, I'll just stick to tailwind. Maybe make some utility classes in the config.